tests/foreman/ui/test_operatingsystem.py

```diff
@@ -34,7 +34,8 @@ def test_positive_end_to_end(session, module_org, module_location, target_sat):
     minor_version = gen_string('numeric', 2)
     description = gen_string('alpha')
     family = 'Red Hat'
-    hash = HASH_TYPE['md5']
+    status = target_sat.execute('fips-mode-setup --check').stdout
```

Comment:
iirc fips-mode-setup was removed in EL10, and it would fail when Foreman will be deployable on EL10 in future, so I'd suggest using checking `/proc/sys/crypto/fips_enabled` for fips status, and we've a `is_fips_enabled` host helper already for this